src/main/java/it/gov/pagopa/reward/notification/dto/InitiativeRefund2StoreDTO.java

```diff
@@ -0,0 +1,29 @@
+package it.gov.pagopa.reward.notification.dto;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.*;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@EqualsAndHashCode
+@Builder
+public class InitiativeRefund2StoreDTO {
+    @JsonProperty("initiativeId")
+    private String initiativeId;
+
+    @JsonProperty("initiativeName")
+    private String initiativeName;
+
+    @JsonProperty("organizationId")
+    private String organizationId;
+
+    @JsonProperty("general")
```

Comment:
add a String field named "organizationVat", which should be stored inside the 
organizationFiscalCode field of RewardNotificationRule (to be added)

ps: put a TODO to remember to check the name (actually is not confirm this name)